geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -497,8 +543,11 @@ public void doTestBigMessageIsFragmented(boolean mcastEnabled, boolean broadcast
 
     // send a big message and expect fragmentation
     GMSMember recipient = broadcastMessage ? null : messenger.localAddress;
+    services.getSerializer().registerDSFID(ByteHolder.DSFID, ByteHolder.class);
     JoinRequestMessage msg = new JoinRequestMessage(recipient, sender,
-        new byte[(int) (services.getConfig().getDistributionConfig().getUdpFragmentSize() * (1.5))],
+        new ByteHolder(
+            new byte[(int) (services.getConfig().getDistributionConfig().getUdpFragmentSize()
+                * (1.2))]),
```

Comment:
Just curious - why the change from 1.5 to 1.2?

Reply:
1.5 was overkill with ByteHolder.  I think I made that change during debugging and just left it.